### PR TITLE
Bugfix/inst 1982 -  Fix bash backspace,arrow issue, top, ps and updated curl with https

### PR DESCRIPTION
**Pull request by srinivasgowda097**

### CODE REVIEW

**Review by shahiinn — DISMISSED** [review]

**Review by amithins — DISMISSED** [review]

File: base/testdata/debug.yaml [hunk 1]
```diff
@@ -11,11 +11,19 @@ commandTests:
...
     command: ["curl","-h"]
     expectedOutput: ['Usage: curl \[options\.\.\.\] <url>']
+  # Check top cmd
+  - name: top
+    command: ["top", "-n", "1", "-b"]
+    exitCode: 0
+  # Check ps cmd
+  - name: ps
+    command: ["ps"]
+    exitCode: 0
```
Comment (eldada) [inline base/testdata/debug.yaml]: Add newline.

File: WORKSPACE [hunk 2]
```diff
@@ -33,22 +33,21 @@ package_manager_repositories()
...
 dpkg_src(
     name = "debian_buster",
     arch = "amd64",
     distro = "buster",
-    sha256 = "996001ca07a8ea54c4d54539de7ed96d1d2cb1a2c0a3040c1be8fa89820f6bff",
-    snapshot = "20190224T095432Z",
-    url = "https://deepscan.jfrog.io/deepscan/debian/archive",
+    sha256 = "15d6bbf761c95c4795d50d41a0385fd643c32b670c58d7abaa17f9736f17e578",
+    snapshot = "20210122T024434Z",
+    url = "https://snapshot.debian.org/archive",
```
Comment (eldada) [inline WORKSPACE:43]: Why are we not taking it from our servers anymore?

File: locale/BUILD [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package(default_visibility = ["//visibility:public"])
+
+sh_binary(
+    name = "extract_locale",
+    srcs = ["extract.sh"],
+)
```
Comment (eldada) [inline locale/BUILD]: Add a new line

File: locale/extract.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+rm data.tar.xz
+
+# On Mac OS, you need to make sure that GNU Tar is used instead of the BSD tar that is shipped with MacOS.
+# To resolve this you can install gnu-tar via Homebrew and symlink it as /usr/local/bin/tar.
+tar -c --owner=0 --group=0 -f locale.tar ./usr
+
+mv locale.tar "$OUT_TAR"
+
+rm -rf usr
```
Comment (eldada) [inline locale/extract.sh]: Add a new line

File: locale/locale.bzl [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            executable = True,
+            allow_files = True,
+        ),
+    },
+    executable = False,
+    outputs = {
+        "tar": "%{name}.tar",
+    },
+    implementation = _impl,
+)
```
Comment (eldada) [inline locale/locale.bzl]: Add a new line

File: base/testdata/debug.yaml (hunk 1 above)
Comment (srinivasgowda097) [reply base/testdata/debug.yaml]: added

File: locale/BUILD (hunk 3 above)
Comment (srinivasgowda097) [reply locale/BUILD]: added

File: locale/extract.sh (hunk 4 above)
Comment (srinivasgowda097) [reply locale/extract.sh]: added

File: locale/locale.bzl (hunk 5 above)
Comment (srinivasgowda097) [reply locale/locale.bzl]: added

**Review by amithins — APPROVED** [review]

**Review by rahulsadanandan — APPROVED** [review]

**Review by kammathavaran — APPROVED** [review]

**Pull request merged**